src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -146,6 +146,30 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             }
         }
 
+        if (const auto& child{ GetTemplateChild(L"Expander") })

```

Comment:
We should probably reorder these to be...
```c++
if (/*we have an Expander*/) {/*apply Auto Props to expander control*/}
else if (/*we have Content*/) { /*apply Auto Props to content*/ }
```
That way we aren't redundantly applying the Auto Props.